app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesis.js

```diff
@@ -30,12 +29,10 @@ const Title = ({ el, counter, molSerials }) => {
       ? [...title, <span key={key}>{smn} (<b>{us}</b>)</span>, comma]
       : [...title, <span key={key}>&quot;<b>NAME</b>&quot; (<b>{us}</b>)</span>, comma];
   });
-  title = _.flatten(title).slice(0, -1);
+  title = title.slice(0, -1);
```

Comment:
Why did you remove `_.flatten`?
I don't know which kind of title could occur. But if you have an array in an array the result could be different

Reply:
We create the `title` in line 22 and only ever "append" (i.e. `[…title, ...]`) two `<span>` Tags to it. So there are no nested arrays.